app/params/builder.go

```diff
@@ -10,7 +10,7 @@ func SetBuilderDefaultConfig() {
 	sdk.SetAddrCacheEnabled(false)
 	defer sdk.SetAddrCacheEnabled(true)
 
-	buildertypes.DefaultEscrowAccountAddress = authtypes.NewModuleAddress(buildertypes.ModuleName).String()
+	buildertypes.DefaultEscrowAccountAddress = authtypes.NewModuleAddress(buildertypes.ModuleName)
```

Comment:
since `.String()` is not used, can we remove `sdk.SetAddrCacheEnabled(false)` ?

Reply:
Good catch